Sources/AWSLambdaRuntimeCore/LambdaRunner.swift

```diff
@@ -43,7 +43,9 @@ internal final class LambdaRunner {
             allocator: self.allocator,
             terminator: terminator
         )
-        return Handler.makeHandler(context: context)
+
+        return handlerType.makeHandler(context: context)
+            .map { $0 as any ByteBufferLambdaHandler }
```

Comment:
don't drop types here.

Reply:
this was required to make the type system happy. do you have a better way to do this?

Reply:
we don't need to drop here, if we keep the old syntax above.